java/ql/src/semmle/code/java/security/RequestForgeryConfig.qll

```diff
@@ -0,0 +1,29 @@
+/**
+ * Provides a taint-tracking configuration characterising request-forgery risks.
```

Comment:
We've until now tried to avoid putting configurations belonging to queries in qll-files. I can see the desire to do so to easily support inline test expectation - in other places we've just copied the config to the test, so we could consider to do that as well here (and share the source definition as `RequestForgerySource` in `RequestForgery.qll`). If we keep things as-is, then we should at the very least put a big fat warning in the qldoc here that this file isn't to be imported in any `.qll` file - all imports should be directly in `.ql` files.

Reply:
```suggestion
 * Provides a taint-tracking configuration characterising request-forgery risks.
 *
 * Only import this directly from .ql files, to avoid the possibility of polluting the Configuration hierarchy accidentally.
```

Reply:
Went with the second option since the configuration isn't trivial, and the chance of the test and the query accidentally diverging in future seems non-negligible